src/Analyzer.JsonRuleEngine.UnitTests/InOperatorTests.cs

```diff
@@ -39,7 +38,7 @@ public void EvaluateExpression(bool evaluationResult, object desiredValue, param
         [TestMethod]
         public void GetName_ReturnsCorrectName()
         {
-            Assert.AreEqual("In", new InOperator(new JObject(), false).Name);
+            Assert.AreEqual("In", new InOperator(new JArray(), false).Name);
         }
     }
```

Comment:
Also add a test for evaluating a null JToken.  This is different than evaluating a JToken for which the value in it is null.
For example, something like this:
``` diff
- var desiredValueJToken = TestUtilities.ToJToken(null);
+ var desiredValueJToken = null;
inOperator.EvaluateExpression(desiredValueJToken)
```

This will probably need to be a test on its own, since it doesn't fit what other tests are doing.

Reply:
I can't compile `inOperator.EvaluateExpresion(null)`: _Cannot assign <null> to an implicitly typed variable_

Reply:
Whoops - `JToken desiredValueJToken = null;` 🙂